lib/middleware.ts

```diff
@@ -189,5 +278,17 @@ var formatLine = (options_arg) => {
 if (process.env.NODE_ENV === "test") {
   module.exports = (clever_options, morgan_options) => morgan(formatLine(clever_options), morgan_options);
 } else {
-  module.exports = (clever_options) => morgan(formatLine(clever_options), {stream: process.stderr});
+  module.exports = (clever_options, context_logger_options = defaultContextLoggerOpts) => {
```

Comment:
not sure if this should be shared between the middleware and this package or not.